lib/Doctrine/DBAL/Platforms/Keywords/KeywordList.php

```diff
@@ -8,6 +8,8 @@
 
 /**
  * Abstract interface for a SQL reserved keyword dictionary.
+ *
+ * @psalm-consistent-constructor
```

Comment:
This could be removed in `4.0.x` thanks to https://github.com/doctrine/dbal/pull/4583.